packages/rx_bloc_cli/lib/src/models/configurations/auth_configuration.dart

```diff
@@ -6,7 +6,7 @@ class AuthConfiguration {
     required this.socialLoginsEnabled,
     required this.otpEnabled,
     required this.pinCodeEnabled,
-    required this.authMatrixEnabled,
+    required this.authMFA,
```

Comment:
This could be simplified to just `mfa`/`MFA`, unless there's a reason to keep it this way.

Reply:
Good catch. It has been renamed to `mfaEnabled`